drivers/src/qemuexit.rs

```diff
@@ -0,0 +1,35 @@
+use qemu_exit;
+use qemu_exit::QEMUExit;
+
+use crate::Driver;
+
+pub struct QemuExit {
+    address: usize,
+    #[cfg(target_arch = "riscv64")]
+    inner: qemu_exit::RISCV64,
+}
+
+impl QemuExit {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> QemuExit {
+        #[cfg(target_arch = "riscv64")]
+        Self {
+            address: 0x100000,
+            inner: qemu_exit::RISCV64::new(0x100000),
```

Comment:
```suggestion
        #[cfg(target_arch = "riscv64")]
        Self {
            address: 0x100000,
            inner: qemu_exit::RISCV64::new(0x100000),
```
```suggestion
        Self {
            address: 0x100000,
            #[cfg(target_arch = "riscv64")]
            inner: qemu_exit::RISCV64::new(0x100000),
```

Or was that done on purpose?